test/integration/helpers.coffee

```diff
@@ -62,14 +62,19 @@ createServer = ->
 
   app.use(bodyParser.json({size: '5mb'}))
   app.use((req, res, next) ->
+    serverRuntimeInfo.requestedOnce = not serverRuntimeInfo.requested
```

Comment:
I don't understand the naming - `once` IMHO tells me, it was done exactly 1 time. But it looks like it's just a negation of `serverRuntimeInfo.requested`.